demo/MMSegmentation_Tutorial.ipynb

```diff
@@ -156,8 +157,8 @@
    },
    "outputs": [],
    "source": [
-    "config_file = '../configs/pspnet/pspnet_r50-d8_512x1024_40k_cityscapes.py'\n",
-    "checkpoint_file = '../checkpoints/pspnet_r50-d8_512x1024_40k_cityscapes_20200605_003338-2966598c.pth'"
+    "config_file = '../mmsegmentation/configs/pspnet/pspnet_r50-d8_512x1024_40k_cityscapes.py'\n",
+    "checkpoint_file = '../mmsegmentation/checkpoints/pspnet_r50-d8_512x1024_40k_cityscapes_20200605_003338-2966598c.pth'\n"
```

Comment:
as this demo is in `mmsegmentation/demo/` and config file is in `mmsegmentation/configs`, we think `mmsegmentation` is not necssary.
```suggestion
    "config_file = '../configs/pspnet/pspnet_r50-d8_512x1024_40k_cityscapes.py'\n",
    "checkpoint_file = '../checkpoints/pspnet_r50-d8_512x1024_40k_cityscapes_20200605_003338-2966598c.pth'\n"
```

Reply:
> Many thanks for your contribution and please sign CLA first.
> 
> After our discussion, we think `keep 2 notebooks` you suggested is a good idea, and this notebook is just for users running locally, so it might only fix mmcv installation in this PR, and keep the other as it is. thanks again

yeah i agree, 2 notebooks will work best since theyre designed to run in 2 completely different enviroments, i have signed the cla